src/routes/(app)/strafbakken/+page.server.ts

```diff
@@ -0,0 +1,51 @@
+import type { Actions, PageServerLoad } from "./$types";
+import db from "$lib/server/db";
+import { getUser } from "$lib/server/userCache";
+import { fail } from "@sveltejs/kit";
+
+// Load een overview van alle strafbakken
+export const load = (async () => {
+  return {
+    strafbakken: await db.user.findMany({
+      select: {
+        firstName: true,
+        nickname: true,
+        id: true,
+        _count: {
+          select: {
+            StrafbakReceived: {
+              where: {
+                dateDeleted: null,
```

Comment:
waarom gebruik je hier dateDeleted? Wordt een strafbak verwijderd als ie getrokken is?

Reply:
Das namelijk niet heel nice, dan heb je geen overzicht van de getrokken bakken meer

Reply:
nee dat is de dag dat hij getrokken is, het heet alleen deleted maar er wordt niks gedelete